core-services/image-mirroring/apiserver-network-proxy/OWNERS

```diff
@@ -0,0 +1,18 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
+# Fetched from https://github.com/openshift/apiserver-network-proxy root OWNERS
+# If the repo had OWNERS_ALIASES then the aliases were expanded
+# Logins who are not members of 'openshift' organization were filtered out
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- csrwng
+- enxebre
+- ironcladlou
+- sjenning
+options: {}
+reviewers:
+- csrwng
+- enxebre
+- ironcladlou
+- relyt0925
+- sjenning
```

Comment:
these mirror files/directories are based on the repo org, so i'd expect this to be "image-mirroring/hypershift", given where it is pushing to, and that already exists.  Why not add it there?